napari/layers/_labels_layer/model.py

```diff
@@ -544,6 +544,7 @@ def _update_thumbnail(self):
         downsampled = self.raw_to_displayed(downsampled)
         colormapped = self.colormap.map(downsampled)
         colormapped = colormapped.reshape(downsampled.shape + (4,))
+        colormapped[..., 3] = 1
```

Comment:
what was happening here before?

Reply:
space where there were no shapes was being rendered as transparent instead of black, i think rendering as black makes it more clear what is going on. i've added a comment